zerver/tests/test_signup.py

```diff
@@ -80,7 +80,7 @@ def test_public_urls(self):
                           "/en/accounts/login/", "/ru/accounts/login/",
                           "/help/"],
                     302: ["/", "/en/", "/ru/"],
-                    401: ["/json/streams/Denmark/members",
+                    401: ["/json/streams/1/members",
```

Comment:
We might to get the stream id of Denmark in the test code, just in case populate_db.py ever changes here.